packages/server-boilerplate/src/connections/CachedOutboundConnection.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { CacheContainer } from 'node-ts-cache';
+import { MemoryStorage } from 'node-ts-cache-storage-memory';
```

Comment:
Decided to go with an external library for the caching mechanism? Overkill perhaps? 

Originally I wanted to use the js decorators that this library provided for a really simple interface, but they proved a tad awkward when being used in a class like this, and it seems we'd have to play with our package.json settings anyway to use decorations since they're still experimental...